flake.nix

```diff
@@ -11,31 +11,44 @@
           pkgs = import nixpkgs {
             inherit system overlays;
           };
+          gohip-vendor-hash = import ./gohip.vendor.hash.nix;
           gohip-package = (pkgs.buildGoModule {
             name = "gohip";
             pname = "gohip";
 
             src = ./.;
-            vendorHash = "sha256-jZcR+OMiGplA5yVMpQT4qNKH+tyFp7PXQVoG+oghBLs=";
+            vendorHash = gohip-vendor-hash pkgs;
             proxyVendor = true;
             excludedPackages = ["osdata" "others" "systemd" "types"];
           });
 
+          upgrade-nix-gohip-package = pkgs.writeShellScriptBin "upgrade-nix-gohip" ''
+            echo "UPGRADING"
+            echo "pkgs : pkgs.lib.fakeHash" > $WORKSPACE/gohip.vendor.hash.nix
+            NEW_HASH=`nix build .#default 2>&1 | grep " got: " | awk '{print $2}'`
```

Comment:
you probably dont want/need `2>&1`, do you ?

Reply:
The actual value I'm trying to extract lives on stderr. I'm binding stdout and stderr together for simplicity